### DISCUSSION TITLE
Date entry improvment. (direct entry and month selection from drop down)

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#22] [Ideas]
The current calendar does not allow direct entry of a month (ie from a drop down), its annoying to click the arrows if one has to go several month back. 
As a super duper option, and since most of the ppl are using dashboard from a computer, I think direct entree of date would be very useful such as mm/dd/yyyy.

**Comment by jlai** [comment]
Yes, direct entry is recommended for screen readers in particular. I'll add it.
